### ISSUE TITLE
IndexError: boolean index did not match indexed array along dimension 0

### ISSUE THREAD

**Issue by cqgg-163** [SCUT-BIP-Lab/PointDC#5]
HI，Using another dataset still results in the same issue.
![error 1](https://github.com/user-attachments/assets/6701e034-489e-4e73-985f-1245877ce415)

**Comment by HalvesChen** [comment]
Which version dataset you use?

**Comment by HalvesChen** [comment]
The data dimensions vary greatly. Is there a problem with the file? Can the file name match?

**Comment by cqgg-163** [comment]
I think I may have found the problem. Let me try again.

**Comment by cqgg-163** [comment]
Thanks,there are no further questions.

**Comment by HalvesChen** [comment]
Can you elaborate on the cause of the problem? I will update my readme.

**Comment by cqgg-163** [comment]
pointdc_mk/data/S3DIS/Stanford3dDataset_v1.2/Area_5/office_19
I forgot the specific details, but the main changes include replacing office_19 with office_18 and removing S3D.S.PY and prepare.py   .numpy（）from S3D.S.PY and prepare.py

**Comment by cqgg-163** [comment]
pointdc_mk/data/S3DIS/Stanford3dDataset_v1.2/Area_5/office_19
I forgot the specific details, but the main changes include replacing office_19 with office_18 and removing .numpy（）from S3D.S.PY and prepare.py
